docs/best_practices/repositories.rst

```diff
@@ -24,8 +24,26 @@ repositories in a structure similar to the following:
 The highest level directory contains only a few folders for the major types of
 Galaxy repositories; tools, packages, data managers, and sometimes visualizations and datatypes.
 
-Packages
---------
+ToolShed Repositories
+---------------------
+
+A Github repository many correspond to any number of published ToolShed
+repositories.
+
+Every unrestricted tool shed repository should contain a README_ file -
```

Comment:
What toolshed documentation? :P I think it has a place here, anyone wanting to follow best practices and uploading their repositories should know these terms.

There will definitely be a LOT of overlap between the various components, which generally screams "this should be in the wiki and linked to".